magento/pwa-studio
# [PWA-840] Localize Mini Cart & [PWA-839] Localize Search

## Description

Translate strings within Mini Cart and Search

## Related Issue





Mini Cart: https://jira.corp.magento.com/browse/PWA-840
Search: https://jira.corp.magento.com/browse/PWA-839

## Acceptance 



### Verification Stakeholders

@dpatil-magento 
@davemacaulay 
@jimbo 
### Specification



### Verification Steps


1. Go to Mini Cart and Search areas of the application.
2. Verify strings are translated where applicable.

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
